### ISSUE TITLE
FastCUT LR schedule.

### ISSUE THREAD

**Issue by tmabraham** [taesungp/contrastive-unpaired-translation#39]
I was looking through the default options, and I noticed that for FastCUT, the default learning rate schedule was 150 epochs at constant LR and 50 epochs decaying the LR to zero. This is opposed to the CycleGAN learning rate schedule (100 epochs at constant LR, 100 epochs decaying), which may contradict with what the appendix of the paper says:
> the fast variant FastCUT is trained up to 200 epochs, following CycleGAN.

Does this learning rate schedule show better results for FastCUT?

**Comment by taesungp** [comment]
For FastCUT, we saw that 150 + 50 showed slightly better results than 100 + 100. What we meant in that sentence is that the total number of epochs is the same for FastCUT and CycleGAN, and hence the comparison between them is fair.

**Comment by ranch-hands** [comment]
> For FastCUT, we saw that 150 + 50 showed slightly better results than 100 + 100. What we meant in that sentence is that the total number of epochs is the same for FastCUT and CycleGAN, and hence the comparison between them is fair.

Hi There
Do you know how we can test the network on other datasets? please help.
Thanks in advance
